client/src/views/Game.vue

```diff
@@ -1,5 +1,56 @@
 <template>
-  <div>
+  <div class="mb-0 game-wrapper">
+    <v-card 
+      class="overflow-hidden"
+    >
+      <!-- Game Players Drawer -->
+      <v-navigation-drawer
+          :expand-on-hover="true"
+          style="min-height: 100%;"
```

Comment:
Can this be moved out to a class? I believe that best practice is to avoid using style's directly whenever possible.

Reply:
ohp. sorry missed that one.

Reply:
we actually didn't need that one anyway.